paramiko/pkey.py

```diff
@@ -281,6 +281,11 @@ def _read_private_key_file(self, tag, filename, password=None):
 
     def _read_private_key(self, tag, f, password=None):
         lines = f.readlines()
+
+        # Support binary file-like objects
+        if type(lines[0]) is bytes:
+            lines = [line.decode('ascii') for line in lines]
```

Comment:
Seems like flake8 is complaining:
```
--- paramiko/pkey.py	2019-10-02 19:40:00.632415 +0000
+++ paramiko/pkey.py	2019-10-02 19:40:32.048944 +0000
@@ -282,11 +282,11 @@
     def _read_private_key(self, tag, f, password=None):
         lines = f.readlines()
 
         # Support binary file-like objects
         if type(lines[0]) is bytes:
-            lines = [line.decode('ascii') for line in lines]
+            lines = [line.decode("ascii") for line in lines]

         start = 0
         beginning_of_key = "-----BEGIN " + tag + " PRIVATE KEY-----"
         while start < len(lines) and lines[start].strip() != beginning_of_key:
             start += 1
would reformat paramiko/pkey.py
```

Reply:
Completely missed that, sorry! I've pushed a fix.